src/main/java/uk/ac/ebi/embl/converter/utils/ConversionEntry.java

```diff
@@ -10,29 +10,95 @@
  */
 package uk.ac.ebi.embl.converter.utils;
 
+import java.io.IOException;
+import java.io.StringReader;
+
 public class ConversionEntry {
     String sOID;
     String sOTerm;
     String definition;
     String feature;
-    String qualifier1;
-    String qualifier2;
+    String[] qualifier1;
+    String[] qualifier2;
 
     ConversionEntry(String[] tokens) {
         this.sOID = tokens[0];
         this.sOTerm = tokens[1];
         this.definition = tokens[2];
         this.feature = tokens[3];
-        for (String token : tokens)
-            if (tokens.length > 4) this.qualifier1 = tokens[4].equalsIgnoreCase("null") ? null : tokens[4];
-        if (tokens.length > 5) this.qualifier2 = tokens[5].equalsIgnoreCase("null") ? null : tokens[5];
+        if (tokens.length > 4) this.qualifier1 = parseQualifier(tokens[4]);
+        if (tokens.length > 5) this.qualifier2 = parseQualifier(tokens[5]);
+    }
+
+    private String[] parseQualifier(String str) {
+        StringReader reader = new StringReader(str);
+        String key = parseQualifierKey(reader);
+        if (key != null) {
+            String value = parseQualifierValue(reader);
+            return new String[] {key, value};
+        }
+        return null;
+    }
+
+    private String parseQualifierKey(StringReader reader) {
+        try {
+            int i;
+            StringBuilder key = new StringBuilder();
+            while ((i = reader.read()) != -1) {
+                char c = (char) i;
+                switch (c) {
+                    case '=' -> {
+                        return key.toString();
+                    }
+                    case '/' -> {
+                        // Ignore first char
+                    }
+                    default -> key.append(c);
+                }
+            }
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        return null;
+    }
+
+    private String parseQualifierValue(StringReader reader) {
+        try {
+            int i;
+            StringBuilder key = new StringBuilder();
+            boolean openQuote = false;
+            boolean scapedCharacter = false;
+            while ((i = reader.read()) != -1) {
+                char c = (char) i;
+                if (scapedCharacter) {
+                    scapedCharacter = false;
+                    key.append(c);
+                    continue;
+                }
+                switch (c) {
+                    case '\\' -> {
+                        scapedCharacter = true;
+                    }
+                    case '"' -> {
+                        if (openQuote) return key.toString();
+                        else openQuote = true;
+                    }
+                    default -> {
+                        key.append(c);
+                    }
+                }
+            }
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        return null;
```

Comment:
Can't we just use String operations to compute the qualifier key and value? It might be easy, I think